docs/content/doc/installation/database-preparation.en-us.md

```diff
@@ -0,0 +1,153 @@
+---
+date: "2019-01-16"
+title: "Database Preparation"
+slug: "database-prep"
+weight: 10
+toc: true
+draft: false
+menu:
+  sidebar:
+    parent: "installation"
+    name: "Database preparation"
+    weight: 20
+    identifier: "database-prep"
+---
+
+You need a database to use Gitea. Gitea supports PostgreSQL, MySQL, SQLite, and MSSQL. This page will guide into preparing database. Only PostgreSQL and MySQL will be covered here since those database systems are widely-used in production.
+
+Database instance can be on same machine as Gitea (local database setup), or on different machine (remote database).
+
+## MySQL
+
+1.  On database instance, login to database console as root:
```

Comment:
Can you add a note for Windows users as well? Or at least mention that these commands are for Linux.